### DISCUSSION TITLE
Override install command with "npm install --force", cause infinite install loop while deploying.

### DISCUSSION THREAD

**Discussion by enrich5588** [vercel/community#5253] [Help]
### Summary

I have a Next.js project need to deploy. But I have some package version doesn't peer, so I need to use --force.
But after I changed my Build & Development Settings and redeploy my project. It fall in an infinite install loop.
Anyone know the reason?

### Steps to Reproduce

![螢幕擷取畫面 2023-12-21 180614](https://github.com/vercel/community/assets/117438128/bc8a0d47-a8f4-49e2-b1d1-eaf8bfe5b789)
![e23edcfa-190f-4d20-9058-722594b83b6d](https://github.com/vercel/community/assets/117438128/f559a40e-c9c2-4380-ba12-aa7302b14886)

**Comment by swarnava** [comment]
Hi there,

If you use `npm install --legacy-peer-deps`, does it fix the issue?

**Reply by enrich5588** [reply]
Same issue
![ebe47733-d85d-4964-8e82-061d1a86e3c9](https://github.com/vercel/community/assets/117438128/49973b74-fbb0-4a61-b0b0-f6224ba09bb7)
